.github/workflows/MERGE_DEVELOP_TO_V2.yml

```diff
@@ -0,0 +1,33 @@
+name: Merge develop to
```

Comment:
Think this should read "Merge develop to v2"

Reply:
yep, fixed both errors